common/transport/coap/src/test/java/org/thingsboard/server/transport/coap/efento/CoapEfentTransportResourceTest.java

```diff
@@ -70,26 +110,88 @@ void checkContinuousSensor() {
         assertThat(efentoMeasurements.get(0).getTs()).isEqualTo(tsInSec * 1000);
         assertThat(efentoMeasurements.get(0).getValues().getAsJsonObject().get("temperature_1").getAsDouble()).isEqualTo(22.3);
         assertThat(efentoMeasurements.get(0).getValues().getAsJsonObject().get("humidity_2").getAsDouble()).isEqualTo(20);
-        assertThat(efentoMeasurements.get(1).getTs()).isEqualTo((tsInSec + 180) * 1000);
+        assertThat(efentoMeasurements.get(1).getTs()).isEqualTo((tsInSec + 180 * 5) * 1000);
         assertThat(efentoMeasurements.get(1).getValues().getAsJsonObject().get("temperature_1").getAsDouble()).isEqualTo(22.4);
         assertThat(efentoMeasurements.get(1).getValues().getAsJsonObject().get("humidity_2").getAsDouble()).isEqualTo(30);
+        checkDefaultMeasurements(measurements, efentoMeasurements,180 * 5, false);
```

Comment:
Please fix code style issues

```suggestion
        checkDefaultMeasurements(measurements, efentoMeasurements, 180 * 5, false);
```